packages/wallet/backend/src/rafiki/rafiki-client.ts

```diff
@@ -163,6 +163,9 @@ export class RafikiClient implements IRafikiClient {
     })
 
     if (!response.withdrawEventLiquidity?.success) {
+      if (response.withdrawEventLiquidity?.message === 'Transfer exists') {
+        return true
+      }
```

Comment:
I think for me me it happened because of running in dev mode (breakpoints and all), not sure. I just noticed it. In case this message gets returned by Rafiki, our code threw an error with the message (BadRequest underneath). The error would have made rafiki retry the request, and a loop would start. This change basically returns 200 to rafiki so it doesn't retry anymore.